core/backend/src/rpc-impl/IModelReadRpcImpl.ts

```diff
@@ -194,6 +194,37 @@ export class IModelReadRpcImpl extends RpcInterface implements IModelReadRpcInte
     return iModelDb.getMassProperties(props);
   }
 
+  public async getMassPropertiesPerCandidate(tokenProps: IModelRpcProps, props: MassPropertiesPerCandidateRequestProps): Promise<MassPropertiesPerCandidateResponseProps[]> {
```

Comment:
Rpc interfaces are `@internal` and shouldn't be called directly by apps (including yours). Provide `IModelConnection.getMassPropertiesPerCandidate`.